usr/src/test/zfs-tests/tests/functional/cli_root/zfs_receive/zfs_receive_010_pos.ksh

```diff
@@ -137,6 +137,11 @@ $TOUCH $mntpnt2/f18
 $RM $mntpnt/h17
 $RM $mntpnt2/h*
 
+# Add empty objects to $fs to exercise dmu_traverse code
+for i in `seq 1 100`; do
```

Comment:
Agreed that if the test suite depends on gnu-coreutils, that should be noted in the packaging.  I think the question is: is that OK, if not why not?